### ISSUE TITLE
Telegram always caches downloads to local drive even if Download Folder is set to removable

### ISSUE THREAD

**Issue by superbonaci** [overtake/TelegramSwift#993]
Looks I reported the issue to wrong app, this should be it.
Original issue which happens to TelegramSwift: https://github.com/telegramdesktop/tdesktop/issues/26555

**Comment by qzhsjz** [comment]
still having this issue.

**Comment by viperadnan-git** [comment]
still having this issue, taking double space

**Comment by vicmarto** [comment]
Same here :(

**Comment by TargiX** [comment]
Yeah, that's weird, I just realized that it took 640gb of my downloaded data on my local drive even If I set it to removable! Anybody planning to work on it?
